src/arrayMethodSort.js

```diff
@@ -3,9 +3,36 @@
 /**
  * Implement method Sort
  */
+const charsCompare = (charA, charB) => {
+  if (('' + charA) > ('' + charB)) {
```

Comment:
You may use .toString method here.
It's more understandable for reading.
But if you want - leave as it is